xesmf/backend.py

```diff
@@ -175,6 +193,7 @@ def esmf_regrid_build(sourcegrid, destgrid, method,
 
         - 'bilinear'
         - 'conservative', **need grid corner information**
+        - 'conservative_normed', **need grid corner information**
```

Comment:
I'm wondering if mapping ESMFpy method *options* to new xESMF *methods* will scale well. 

ESMFPy has these clear and well documented `IntEnum` classes  in `api.constants` for options. To make it more pythonic, maybe xESMF could convert those to string options, e.g. 
```
norm_type : {'dstarea', 'fracarea'}
  The type of normalization used when producing the weights.
```
and then
```
ESMF.Regrid(..., 
norm_type=getattr(NormType, norm_type.upper()),
...)
```